src/Environment/EnvironmentVariables.php

```diff
@@ -97,9 +100,9 @@ public static function fromEnvironment(ImportGpgKeyFromString $importKey): self
      */
     private static function getenv(string $key): string
     {
-        $value = getenv($key);
+        $value = Env\get_var($key);
 
-        Assert::stringNotEmpty($value, sprintf('Could not find a value for environment variable "%s"', $key));
+        Psl\invariant($value !== null && $value !== '', Str\format('Could not find a value for environment variable "%s"', $key));
```

Comment:
@azjezz why not `Type\non_empty_string()->assert($value),` here?

Reply:
we, you can do that too, same thing

Reply:
actually, can't:

```
The declared return type 'non-empty-string' for EnvironmentVariables::getenv is more specific than the inferred return type 'string' (see https://psalm.dev/070)

The type 'string' is more general than the declared return type 'non-empty-string' for Owsy\AwsEcrPasswordFetcher\Environment\EnvironmentVariables::getenv (see https://psalm.dev/129)
        return $value;
```

Reply:
ah, yes it does. i was trying
```php
        Psl\invariant(
            Type\non_empty_string()->assert($value),
            Str\format('Could not find a value for environment variable "%s"', $key)
        );
```
instead of just 
```php
Type\non_empty_string()->assert($value);
```

but i guess you then lose the custom invariant error message.

Reply:
yea, you lose the message with assert, maybe we could add a costume message as a second argument, not sure 🤔

Reply:
Sounds good!